superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts

```diff
@@ -174,6 +175,7 @@ export const countries = {
   slovenia,
   spain,
   sweden,
+  srilanka,
```

Comment:
The linter seems to be complaining about this PR. It _might_ be that this is not quite in alphabetical order. You can have it (hopefully) auto-fix by running `pre-commit run --all-files`

Reply:
yes i will run it and push again.